extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.setup.js

```diff
@@ -184,7 +184,7 @@ angular.module('PaperUI.controllers.setup', []).controller('SetupPageController'
         thingSetupService.add({
             'enableChannels' : !$scope.advancedMode
         }, thing, function() {
-            toastService.showDefaultToast('Thing added');
+            toastService.showDefaultToast('Thing added.', 'Show Thing', 'configuration/things/view/' + thing.UID);
             $scope.navigateTo('setup/search/' + $scope.thingType.UID.split(':')[0]);
```

Comment:
Yes, it is. The actual path in this controller is built on top of /inbox/{path}. So the actual path in above case will be /inbox/setup/search.